CHANGELOG.md

```diff
@@ -68,6 +68,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 * Some headers were cleaned up and now `#include` fewer
   other headers. You might have to add some `#includes`s
   in your code.
+* Added a new build switch `BUILD_BASEMQ` for controlling whether `FairRoot::BaseMQ` and
+  dependent targets are built. It is enabled by default and will now **require**
+  its external package dependencies FairMQ and Boost.
```

Comment:
Can you please spell out the dependent targets?

Reply:
Sure, however, let's first discuss whether we want to adopt the general `BUILD_<lib>` option scheme. Depending on the details this text will change anyways. Because, if we add build switches for - let's say `BUILD_PARMQ` - what will happen, if the user configures `-DBUILD_BASEMQ=OFF -DBUILD_PARMQ=ON`. In FairMQ I have a helper [`fairmq_build_option`](https://github.com/FairRootGroup/FairMQ/blob/master/cmake/FairMQBuildOption.cmake). But it is rather stupid, it is not recursive and will not switch dependencies on or off automatically (but complain, if a dependency is off e.g.). I remember you discussed such details heavily with @fuhlig1 in #1246, but I have not read and followed it in detail.